### PR TITLE
enable modules s3fs, asymmetric key, and samlauth

### PR DESCRIPTION
**Pull request by loganmcdonald-noaa**
## What does this PR do? 🛠️

Installs some needed modules: samuath, asymmetric key, and s3fs (which was previously installed but not enabled) 

## What does the reviewer need to know? 🤔 

Locally, you will need to run an import after this merges/when pulling it (`drush config:import`). It will automatically update config in staging when merged. 

**Steps:**
1. Enter drupal shell (use `make shell` if available)
2. Run `composer install` to install new modules (see composer.json)
3. Run `drush cim -y` to import module configurations, including enabled setting.
4. Run `drush cache:rebuild` to clear cache
5. Confirm modules are installed and enabled by running `drush pm-list --type=module --no-core --status=enabled` or going to [localhost](http://localhost:8080/admin/modules) in your browser

## Screenshots 📸 
Confirm the modules are installed and enabled:
**Before**
image

**After**
image

### CODE REVIEW

File: web/config/system.theme.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 _core:
   default_config_hash: 6lQ55NXM9ysybMQ6NzJj4dtiQ1dAkOYxdDompa-r_kk
-admin: claro
```
Comment (claire-skies-noaa) [inline web/config/system.theme.yml:3]: Oops, I think you changed this by mistake. The admin theme should be Claro. 
We haven't built out our weather theme to support Drupal UI yet. 

**Claro theme**: 
image


**Weather theme** - not quite there yet:
image

**Review by claire-skies-noaa — DISMISSED** [review]

**Review by claire-skies-noaa — COMMENTED** [review]
I do confirm that the s3fs, asymmetric key, and samlauth modules are enabled. See screenshots 

**Before**
image

**After**
image

**Review by greg-does-weather — APPROVED** [review]
## Code Review Checklist - Documentation only

This is an automated comment on every pull request requiring a review. A checked item is either an assertion that I have tested this item or an indication that I have verified it does not apply to this pull request.

## Documentation review
- [x] Scanned for grammar errors
- [x] Relevant audiences are aware of changes (eng for /architecture, product for /product)
- [x] (if applicable) Diagrams are added

In a nutshell, since this only installs and turns on these modules, there's not really anything to document. But created #410 for the future, once we start using these modules.
